sql/mysql-compatibility.md

```diff
@@ -97,6 +97,8 @@ TiDB implements the asynchronous schema changes algorithm in F1. The Data Manipu
 
 TiDB implements an optimistic transaction model. Unlike MySQL, which uses row-level locking to avoid write conflict, in TiDB, the write conflict is checked only in the `commit` process during the execution of the statements like `Update`, `Insert`, `Delete`, and so on.
 
+Similarly, functions such as `GET_LOCK()` and `RELEASE_LOCK()` and statements such as `SELECT .. FOR UPDATE` will not function as expected in TiDB.
```

Comment:
s/will not function as expected in TiDB./in TiDB do not work in the same way as in MySQL.

Reply:
thx